db/test/helper.js

```diff
@@ -421,30 +431,43 @@ exports.testEntityTable = ({
 
       setup(async function() {
         await resetTables();
-        await Promise.all(Object.values(samples).map(
-          sample => {
-            return Entity.create(sample);
-          }));
+
+        if (!notImplemented.includes('entity-functions')) {
+          await Promise.all(Object.values(samples).map(
+            sample => {
+              return Entity.create(sample);
+            }));
+        }
       });
 
-      makeTests();
+      if (notImplemented.includes('entity-functions')) {
+        test('database setup', async function() {});
```

Comment:
Same as previous comment. The test talk about a database upgrade but nothing happens in the test?

Reply:
Same as above.